src/com/owncloud/android/utils/TouchImageViewCustom.java

```diff
@@ -1,13 +1,27 @@
-/*
- * TouchImageView.java
- * By: Michael Ortiz
- * Updated By: Patrick Lackemacher
- * Updated By: Babay88
- * Updated By: @ipsilondev
- * Updated By: hank-cp
- * Updated By: singpolyma
- * -------------------
- * Extends Android ImageView to include pinch zooming, panning, fling and double tap zoom.
+/**
+ *   ownCloud Android client application
```

Comment:
Better move this file to third_parties package and keep it's license and copyright untouched; we made no change on it.